spi/control-plane/contract-spi/src/main/java/org/eclipse/edc/connector/contract/spi/types/offer/ContractOffer.java

```diff
@@ -81,13 +80,13 @@ public String getId() {
     }
 
     @Nullable
-    public URI getProvider() {
```

Comment:
breaking change: the old getters and builder methods should stay and been deprecated

Reply:
But we're not supporting (and can't support) backward compatibility. This feature also never worked. So what's the point?